tests/model_options/test_default_related_name.py

```diff
@@ -25,6 +29,25 @@ def test_default_related_name(self):
         except AttributeError:
             self.fail("Author should have a books relation.")
 
+    def test_default_related_name_in_queryset_lookup(self):
+        try:
+            Author.objects.get(books=self.book)
+        except FieldError:
+            self.fail("Book should have 'books' related name in query function")
+
+    def test_show_deprecated_message_when_model_name_in_queryset_lookup(self):
+        msg = "Using model name %r in queryset lookup " \
+              "when meta.default_related_name %r is set is deprecated " % ("book", u"books")
```

Comment:
no `u''` prefix -- use `from __future__ import unicode_literals` if you need it.